Rakefile

```diff
@@ -7,6 +7,7 @@ Rake::TestTask.new do |t|
   t.libs = ['lib','test']
   t.test_files = Dir.glob("test/**/*_spec.rb").sort
   t.verbose = true
+  t.warning = false
```

Comment:
Will do. I think it was to silence the errors from some other dependencies (`therubyracer`) and make the test results for rational to view.